test/controllers/api/v2/hosts_controller_test.rb

```diff
@@ -621,15 +621,6 @@ def test_create_invalid
     assert_equal hostgroup.root_pass, Host.find_by(:name => hostname).root_pass
   end
 
-  test 'host update shouldnt diassociate from VM' do
-    hostgroup = FactoryBot.create(:hostgroup, :with_environment, :with_subnet, :with_domain, :with_os, :with_compute_resource)
-    host = FactoryBot.create(:host, :hostgroup => hostgroup)
-    put :update, params: { :commit => "Update", :id => host.id, :params => {:compute_resource_id => ""}}, session: set_session_user
```

Comment:
would it make sense to actually keep this test and make sure the `compute_resource_id` isn't overriden on api update?

Reply:
@tbrisker  The test is not working right now, and it makes sense to make sure the compute_resource_id cannot be updated in API, but not part of this fix, I will create a different PR for it.